src/shared/consts.tsx

```diff
@@ -120,6 +120,7 @@ export enum ItemResponseType {
   TouchPractice = 'touchPractice',
   TouchTest = 'touchTest',
   ABTrails = 'ABTrails',
+  PhrasalTemplate = 'phrasalTemplate',
```

Comment:
I can't seem trigger that exact view you took a screenshot of – apparently I'm supposed to be able to find it by doing ⌘+Shift+P and typing `typescript` into the command palette, but nothing comes up for me:

![CleanShot 2024-08-05 at 14 06 19](https://github.com/user-attachments/assets/3de29830-e529-4c3d-9435-d6cedc7747cd)

But after some research I found that it correlates to the `typescript.tsdk` VS Code setting, which I haven't changed from the default:
![CleanShot 2024-08-05 at 13 37 47@2x](https://github.com/user-attachments/assets/131195d4-edca-4a63-b323-ddcc51fcb4ea)

So I guess I'm using VS Code's version in the editor itself, which is the default? Doing so hasn't seemed to pose any obvious issues for me.

Having said that, before I push my work to GitHub, I typically do ⌘+Shift+B then pick `tsc: build - tsconfig.json` from the list, which always uses the version of `tsc` that is defined by the project. This way I'm always sure that it will compile correctly using the same version of TypeScript that CI is using:

![CleanShot 2024-08-05 at 14 11 02@2x](https://github.com/user-attachments/assets/8560dafa-a0ae-4a49-ba63-8b42f13c1be5)

It also reveals the same errors that `react-app-rewired` does during runtime.